pkgs/top-level/all-packages.nix

```diff
@@ -14909,6 +14909,8 @@ with pkgs;
 
   wavrsocvt = callPackage ../applications/misc/audio/wavrsocvt { };
 
+  welle-io = qt5.callPackage ../applications/misc/welle-io { };
```

Comment:
Please use `libsForQt5.callPackage`. It is different from `qt5.callPackage` , which it is not meant for use outside of the definition of Qt.